src/NServiceBus.Core.Tests/Recoverability/DelayedRetryExecutorTests.cs

```diff
@@ -50,15 +50,15 @@ public async Task When_native_delayed_delivery_should_add_delivery_constraint()
         public async Task Should_update_retry_headers_when_present()
         {
             var delayedRetryExecutor = CreateExecutor();
-            var originalHeadersTimestamp = DateTimeOffsetHelper.ToWireFormattedString(new DateTime(2012, 12, 12, 0, 0, 0, DateTimeKind.Utc));
+            var originalHeadersTimestamp = DateTimeOffsetHelper.ToWireFormattedString(new DateTimeOffset(2012, 12, 12, 0, 0, 0, TimeSpan.Zero));
```

Comment:
The original line created a `DateTime` in UTC. Does the new line create a `DateTimeOffset` in UTC?

Reply:
Yeah, that's what TimeZone.Zero gets you.